tests/layer_tests/pytorch_tests/test_aten_to.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import torch
+from pytorch_layer_test_class import PytorchLayerTest
+
+
+class TestAtenTo(PytorchLayerTest):
+    def _prepare_input(self):
+        import numpy as np
+        return (np.random.randn(1, 3).astype(np.float32),)
+
+    def create_model(self, type):
+
+        import torch
+        import torch.nn.functional as F
+
+        class aten_to(torch.nn.Module):
+            def __init__(self, type):
+                super(aten_to, self).__init__()            
+                self.type = type
+
+            def forward(self, x):
+                return x.to(self.type)
+
+        ref_net = None
+
+        return aten_to(type), ref_net
+
+    @pytest.mark.parametrize("type", [torch.uint8, torch.int8, torch.int16, torch.int32, torch.float32, torch.int64])
```

Comment:
Does it cover only translation from `float32` to all these types, but not vice versa?